packages/dashboard/src/components/template/banner/banner.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { Text } from '@deriv/components';
+import Frame from 'Components/elements/frame';
+import { useStores } from 'Stores';
+
+const Banner: React.FC<TBannerProps> = ({ bg_image_url, type, title }) => {
+    const { config_store } = useStores();
+
+    return (
+        <div className='dw-banner'>
+            <div className='dw-banner-frame'>
+                <Frame
+                    src={`${config_store.asset_path}/images/${bg_image_url}`}
+                    alt={bg_image_url.split('/')[bg_image_url.split('/').length - 1]}
+                />
+                <div className='dw-banner__container'>
+                    <Text size='m' color='colored-background' line_height='xs'>
+                        {type}
+                    </Text>
+                    <Text size='xl' weight='bold' color='colored-background' line_height='xs'>
+                        {title}
+                    </Text>
+                </div>
+            </div>
+        </div>
+    );
+};
+
+type TBannerProps = {
+    bg_image_url: string;
+    type: string;
+    title: string;
+};
+
+export default Banner;
```

Comment:
Since we're observing `config_store.asset_path` we should wrap this in `observer`.